presto-main/src/main/java/com/facebook/presto/operator/DriverContext.java

```diff
@@ -474,6 +415,14 @@ public DriverStats getDriverStats()
                 ImmutableList.copyOf(transform(operatorContexts, OperatorContext::getOperatorStats)));
     }
 
+    private long getPeakUserMemory()
+    {
+        if (operatorContexts.isEmpty()) {
+            return 0L;
+        }
+        return Ordering.natural().max(transform(operatorContexts, OperatorContext::getPeakUserMemory));
```

Comment:
Driver peak memory is not the max of operator peak memory. Driver peak is guaranteed to be somewhere between the max of operator peak memory and the sum of operator peak memory. We probably don't have enough information here to answer the question of `getPeakUserMemory`．